### ISSUE TITLE
Could not create the application Skylight::Util::HTTP::ErrorResponse:

### ISSUE THREAD

**Issue by przbadu** [skylightio/skylight-ruby#96]
I am trying to setup Skylight for my new app, and getting `OpenSSL` error

```sh
bundle exec skylight setup [my-skylight-key]
```

```sh
Could not create the application
Skylight::Util::HTTP::ErrorResponse: #<struct Skylight::Util::HTTP::ErrorResponse request=#<Net::HTTP::Post POST>, exception=#<Skylight::Util::HTTP::StartErro
r: #<OpenSSL::SSL::SSLError: SSL_connect returned=1 errno=0 state=error: certificate verify failed>>>
```

**Comment by wagenet** [comment]
@przbadu Are you on 1.2.2?

**Comment by przbadu** [comment]
I am using `0.10.5`

```sh
➜  riverwise-server git:(master) ✗ gem list skylight

*** LOCAL GEMS ***

skylight (0.10.5)
```

**Comment by przbadu** [comment]
Upgraded to `skylight-1.2.2` and still getting same issue

**Comment by przbadu** [comment]
I am even getting this in `chrome` browser, not sure if this is only for me?


![screen shot 2017-05-04 at 11 11 47 am](https://cloud.githubusercontent.com/assets/4189129/25691447/b5dafc62-30ba-11e7-988b-186d60e6b623.PNG)

**Comment by przbadu** [comment]
Anyways, able to create new app from another system.

**Comment by wagenet** [comment]
@przbadu another solution for that system is to try to update root SSL certificates.

**Comment by przbadu** [comment]
@wagenet thanks, I was thinking about it, I will update ssl and try :)

**Comment by jacobo** [comment]
Getting this same error. How do you "update root SSL certificates"?

**Comment by jacobo** [comment]
apparently the workaround is `SKYLIGHT_FORCE_OWN_CERTS=1`

**Comment by lbaillie** [comment]
The SSL error is due to outdated SSL certs so it's best to update those if possible. If you are using RVM on a Mac OS, you can update your SSL certs by running `rvm osx-ssl-certs update all`. Otherwise yes, you can also set `SKYLIGHT_FORCE_OWN_CERTS=1` and that will also work. 

The version of the gem released this morning (1.3.1) includes information about how to troubleshoot this in the error itself, so hopefully that will be more helpful for people in the future!

**Comment by eatoncw** [comment]
Just to add for anybody needing to set `SKYLIGHT_FORCE_OWN_CERTS=1`, and `rvm osx-ssl-certs update all` not fixing your ssl issue for some reason, the full command to run is
`SKYLIGHT_FORCE_OWN_CERTS=1 bundle exec skylight setup <your setup token>`.

The troubleshooting help only states `Alternatively, try setting SKYLIGHT_FORCE_OWN_CERTS=1 in your environment` (and this was not explicit enough for those easily confused like me!)
